src/templates/career.tsx

```diff
@@ -132,7 +132,7 @@ const CareerPage = ({
           <GridItem xs={12}>
             <Trans i18nKey={'career.action.alternative'}>
               <InfoText>
-                Oder schicke deine Bewerbung an:{' '}
+                Oder schicke deine Bewerbung an:
```

Comment:
Good catch. It seems like this is indeed not needed anymore 👌 
This seems to have been an alternative solution to the `&nbsp;` that you proposed. Maybe you were wondering why you were able to remove this here without breaking the whitespace but at the same time had to introduce it for the contact form?
Changes like this should ring an alarm bell 🔔

Reply:
Thanks for the comment. I think I found the reason for this. The right translation is now used in the contact component. I changed the name of the german translation